.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+**/.DS_Store
```

Comment:
`*.DS_Store` should be enough
I just found this handy overview: https://www.atlassian.com/git/tutorials/saving-changes/gitignore

Reply:
ok good to know. i updated again